### DISCUSSION TITLE
Hello, I have a problem. I hope you can help me

### DISCUSSION THREAD

**Discussion by 756616071** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2676] [Q&A] [answered]
Hello, I have a question to ask you. It is a problem caused by calling the time selection control in your project. I copied the code directly from your demo, but it can't run. This project is newly created directly and there is no other reference

![image](https://user-images.githubusercontent.com/44856671/93233776-0e080c00-f7ae-11ea-8590-6d144ec92090.png)

![image](https://user-images.githubusercontent.com/44856671/93233999-558e9800-f7ae-11ea-8547-6821bfe2b39b.png)
![image](https://user-images.githubusercontent.com/44856671/93234240-a1414180-f7ae-11ea-8778-894de30935fd.png)

**Comment by Keboo** [comment, marked as answer]
Hi @756616071 The demo app creates event handlers that require you to create methods in the code behind. I suspect you just need to remove these two lines:
```xaml
materialDesign:DialogHost.DialogOpenedAttached="CalendarDialogOpenedEventHandler"
materialDesign:DialogHost.DialogClosingAttached="CalendarDialogClosingEventHandler"
```

**Comment by 756616071** [comment]
Hello, I'm glad you can solve the problem. I can open the time selector normally, but I have encountered a new problem. I hope you can help me. How can I know the time I chose and the time I chose to transfer in the second time

**Comment by MichelMichels** [comment]
@756616071 You'll have to bind it to a DateTime property on your ViewModel or get it in code-behind
